diff --git a/internal/types.d.mts b/internal/types.d.mts
index 7e7356b2a6491de4ab7df61e4289ae2878171910..1ecce9b3f76fa99677960f546abb72b0232a3e85 100644
--- a/internal/types.d.mts
+++ b/internal/types.d.mts
@@ -46,15 +46,15 @@ type OverloadedParameters<T> = T extends ({
  * [1]: https://www.typescriptlang.org/tsconfig/#typeAcquisition
  */
 // @ts-ignore
-type UndiciTypesRequestInit = NotAny<import('undici-types').RequestInit> | NotAny<import('undici-types').RequestInit> | NotAny<import("../../../node_modules/undici-types/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../node_modules/undici-types/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../../node_modules/undici-types/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../../../node_modules/undici-types/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../../../../node_modules/undici-types/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../../../../../node_modules/undici-types/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../../../../../../node_modules/undici-types/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../../../../../../../node_modules/undici-types/index.d.ts.mjs").RequestInit>;
+type UndiciTypesRequestInit = NotAny<import('undici-types').RequestInit> | NotAny<import('undici-types').RequestInit>;
 // @ts-ignore
-type UndiciRequestInit = NotAny<import('undici').RequestInit> | NotAny<import('undici').RequestInit> | NotAny<import("../../../node_modules/undici/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../node_modules/undici/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../../node_modules/undici/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../../../node_modules/undici/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../../../../node_modules/undici/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../../../../../node_modules/undici/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../../../../../../node_modules/undici/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../../../../../../../node_modules/undici/index.d.ts.mjs").RequestInit>;
+type UndiciRequestInit = NotAny<import('undici').RequestInit> | NotAny<import('undici').RequestInit>;
 // @ts-ignore
 type BunRequestInit = globalThis.FetchRequestInit;
 // @ts-ignore
-type NodeFetch2RequestInit = NotAny<import('node-fetch').RequestInit> | NotAny<import('node-fetch').RequestInit> | NotAny<import("../../../node_modules/@types/node-fetch/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../node_modules/@types/node-fetch/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../../node_modules/@types/node-fetch/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../../../node_modules/@types/node-fetch/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../../../../node_modules/@types/node-fetch/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../../../../../node_modules/@types/node-fetch/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../../../../../../node_modules/@types/node-fetch/index.d.ts.mjs").RequestInit> | NotAny<import("../../../../../../../../../../node_modules/@types/node-fetch/index.d.ts.mjs").RequestInit>;
+type NodeFetch2RequestInit = NotAny<import('node-fetch').RequestInit> | NotAny<import('node-fetch').RequestInit>;
 // @ts-ignore
-type NodeFetch3RequestInit = NotAny<import('node-fetch.js').RequestInit> | NotAny<import('node-fetch.js').RequestInit> | NotAny<import("../../../node_modules/node-fetch.mjs").RequestInit> | NotAny<import("../../../../node_modules/node-fetch.mjs").RequestInit> | NotAny<import("../../../../../node_modules/node-fetch.mjs").RequestInit> | NotAny<import("../../../../../../node_modules/node-fetch.mjs").RequestInit> | NotAny<import("../../../../../../../node_modules/node-fetch.mjs").RequestInit> | NotAny<import("../../../../../../../../node_modules/node-fetch.mjs").RequestInit> | NotAny<import("../../../../../../../../../node_modules/node-fetch.mjs").RequestInit> | NotAny<import("../../../../../../../../../../node_modules/node-fetch.mjs").RequestInit>;
+type NodeFetch3RequestInit = NotAny<import('node-fetch.js').RequestInit> | NotAny<import('node-fetch.js').RequestInit>;
 // @ts-ignore
 type FetchRequestInit = NonNullable<OverloadedParameters<typeof fetch>[1]>;
 type RequestInits = NotAny<UndiciTypesRequestInit> | NotAny<UndiciRequestInit> | NotAny<BunRequestInit> | NotAny<NodeFetch2RequestInit> | NotAny<NodeFetch3RequestInit> | NotAny<RequestInit> | NotAny<FetchRequestInit>;
diff --git a/internal/types.d.ts b/internal/types.d.ts
index 919ed297e51e870629bfd9d24e79c7d9cd87c771..315bbebf133d9b464c00e9ac9969b477a6310672 100644
--- a/internal/types.d.ts
+++ b/internal/types.d.ts
@@ -46,15 +46,15 @@ type OverloadedParameters<T> = T extends ({
  * [1]: https://www.typescriptlang.org/tsconfig/#typeAcquisition
  */
 // @ts-ignore
-type UndiciTypesRequestInit = NotAny<import('undici-types').RequestInit> | NotAny<import('undici-types').RequestInit> | NotAny<import("../../../node_modules/undici-types/index.d.ts.js").RequestInit> | NotAny<import("../../../../node_modules/undici-types/index.d.ts.js").RequestInit> | NotAny<import("../../../../../node_modules/undici-types/index.d.ts.js").RequestInit> | NotAny<import("../../../../../../node_modules/undici-types/index.d.ts.js").RequestInit> | NotAny<import("../../../../../../../node_modules/undici-types/index.d.ts.js").RequestInit> | NotAny<import("../../../../../../../../node_modules/undici-types/index.d.ts.js").RequestInit> | NotAny<import("../../../../../../../../../node_modules/undici-types/index.d.ts.js").RequestInit> | NotAny<import("../../../../../../../../../../node_modules/undici-types/index.d.ts.js").RequestInit>;
+type UndiciTypesRequestInit = NotAny<import('undici-types').RequestInit> | NotAny<import('undici-types').RequestInit>;
 // @ts-ignore
-type UndiciRequestInit = NotAny<import('undici').RequestInit> | NotAny<import('undici').RequestInit> | NotAny<import("../../../node_modules/undici/index.d.ts.js").RequestInit> | NotAny<import("../../../../node_modules/undici/index.d.ts.js").RequestInit> | NotAny<import("../../../../../node_modules/undici/index.d.ts.js").RequestInit> | NotAny<import("../../../../../../node_modules/undici/index.d.ts.js").RequestInit> | NotAny<import("../../../../../../../node_modules/undici/index.d.ts.js").RequestInit> | NotAny<import("../../../../../../../../node_modules/undici/index.d.ts.js").RequestInit> | NotAny<import("../../../../../../../../../node_modules/undici/index.d.ts.js").RequestInit> | NotAny<import("../../../../../../../../../../node_modules/undici/index.d.ts.js").RequestInit>;
+type UndiciRequestInit = NotAny<import('undici').RequestInit> | NotAny<import('undici').RequestInit>;
 // @ts-ignore
 type BunRequestInit = globalThis.FetchRequestInit;
 // @ts-ignore
-type NodeFetch2RequestInit = NotAny<import('node-fetch').RequestInit> | NotAny<import('node-fetch').RequestInit> | NotAny<import("../../../node_modules/@types/node-fetch/index.d.ts.js").RequestInit> | NotAny<import("../../../../node_modules/@types/node-fetch/index.d.ts.js").RequestInit> | NotAny<import("../../../../../node_modules/@types/node-fetch/index.d.ts.js").RequestInit> | NotAny<import("../../../../../../node_modules/@types/node-fetch/index.d.ts.js").RequestInit> | NotAny<import("../../../../../../../node_modules/@types/node-fetch/index.d.ts.js").RequestInit> | NotAny<import("../../../../../../../../node_modules/@types/node-fetch/index.d.ts.js").RequestInit> | NotAny<import("../../../../../../../../../node_modules/@types/node-fetch/index.d.ts.js").RequestInit> | NotAny<import("../../../../../../../../../../node_modules/@types/node-fetch/index.d.ts.js").RequestInit>;
+type NodeFetch2RequestInit = NotAny<import('node-fetch').RequestInit> | NotAny<import('node-fetch').RequestInit>;
 // @ts-ignore
-type NodeFetch3RequestInit = NotAny<import('node-fetch.js').RequestInit> | NotAny<import('node-fetch.js').RequestInit> | NotAny<import("../../../node_modules/node-fetch.js").RequestInit> | NotAny<import("../../../../node_modules/node-fetch.js").RequestInit> | NotAny<import("../../../../../node_modules/node-fetch.js").RequestInit> | NotAny<import("../../../../../../node_modules/node-fetch.js").RequestInit> | NotAny<import("../../../../../../../node_modules/node-fetch.js").RequestInit> | NotAny<import("../../../../../../../../node_modules/node-fetch.js").RequestInit> | NotAny<import("../../../../../../../../../node_modules/node-fetch.js").RequestInit> | NotAny<import("../../../../../../../../../../node_modules/node-fetch.js").RequestInit>;
+type NodeFetch3RequestInit = NotAny<import('node-fetch.js').RequestInit> | NotAny<import('node-fetch.js').RequestInit>;
 // @ts-ignore
 type FetchRequestInit = NonNullable<OverloadedParameters<typeof fetch>[1]>;
 type RequestInits = NotAny<UndiciTypesRequestInit> | NotAny<UndiciRequestInit> | NotAny<BunRequestInit> | NotAny<NodeFetch2RequestInit> | NotAny<NodeFetch3RequestInit> | NotAny<RequestInit> | NotAny<FetchRequestInit>;
